webapp/lib/view.dart

```diff
@@ -457,10 +457,29 @@ class ContentView {
           titleText: '$driverName',
           datasetLabels: List.filled(0, '', growable: true)
         );
+        driverChart.yUpperLimitRangeSlider.children[1].onChange.listen((e) {
+          var slider = (e.currentTarget as RangeInputElement);
+          var sliderIndicator = (e.currentTarget as Element).previousElementSibling;
+          if (int.parse(slider.value) == 0) {
```

Comment:
At the moment it's a bit confusing the first time it loads that the slider is set to 0, but the chart is set to max. Would it be possible to have the slider start at max by default?

Reply:
Actually that makes a lot of sense. I've refactored this so that it starts at max.